data/meson.build

```diff
@@ -0,0 +1,31 @@
+
+icon_size = ['48', '64']
+
+foreach i : icon_size
```

Comment:
instead of creating a variable and doing a foreach and all of that. You could just do this

```meson
install_subdir (
  'icons',
  install_dir: join_paths ( get_option ('datadir'), 'icons', 'hicolor'),
  strip_directory: true
)
```
and renaming the icons directory from `icons/48` to `icons/48x48/apps`

Reply:
Actually I followed the way of elementary developers. I think it's better to keep it like this for more clarity.